src/qibolab/channels.py

```diff
@@ -22,102 +23,92 @@ class Channel:
 
     name: str
     """Name of the channel from the lab schematics."""
-
-    qubit: Optional["Qubit"] = field(default=None, repr=False)
-    """Qubit connected to this channel.
-    Used to read the sweetspot and filters for flux channels only. ``None`` for non-flux channels.
-    """
-    ports: List[tuple] = field(default_factory=list)
-    """List of tuples (controller, port) connected to this channel."""
-    _local_oscillator: Optional[LocalOscillator] = None
+    port: Optional[Port] = None
+    """Instrument port that is connected to this channel."""
```

Comment:
Are you sure that one `Port` is enough? And also that is even optional?

If I remember correctly, during our discussion with @aorgazf it has been mentioned that you may have one or two ports per-channel.

Reply:
> If having multiple ports in the same channel appears more often, we could lift it to the public interface (for example change `port: Optional[Port]` to `ports: List[Port]`. But if it's only a QM thing, I would leave it in the driver to keep the abstract interface simpler.

I agree on the approach and the resolution :)

> TWPA channels are only connected to local oscillators (Rohde&Schwarz / Erasynth) for which we do not implement ports. My idea was to only implement ports for `Controller` instruments. In principle we could also implement them for `LocalOscillator`s and I believe in this case we could make the `port` not optional, but it would make the LO drivers more complicated (two objects instead of one) and is not absolutely required.

This is mostly a problem of intuition: I was thinking to a `Port` as a physical one. If you say that the physical one is there, but it is not needed at any level, so you'd rather not represent it (for simplicity), I agree with you.